### DISCUSSION TITLE
ipxe not working for rhel 8.6

### DISCUSSION THREAD

**Discussion by Sak0808** [ipxe/ipxe#833] [Q&A]
We are trying to Configure PXE Network Boot Server for RHel 8.6 
But after booting from network we are getting this error:

A start job is running for dev-mapper-live\x2drw.device

Whereas, when we are trying to boot other Centos we are able to boot via /LiveOS/squashfs.img

In RedHat is there a similar path?

**Comment by NiKiZe** [comment]
I don't see any error?
But for net booting, check your distros documentation.

**Reply by stappersg** [reply]
and https://github.com/ipxe/ipxe/discussions/830
